tests/test_route.py

```diff
@@ -0,0 +1,56 @@
+from urllib.request import urlopen
+
+import pytest
+from app.create_app import create_app
+from flask import url_for
+
+
+@pytest.fixture(scope="session")
+def app():
+    app = create_app()
+    return app
+
+
+@pytest.mark.usefixtures("live_server")
+class TestSearchPage:
+    """
+    GIVEN class checks if route for send_notification
+    is up & running
+    """
+
+    def test_notification_route_response(self):
+        url = url_for("notification_bp.send_notification", _external=True)
+        res = urlopen(url)
+        assert res.code == 200
+
+
+@pytest.fixture()
+def flask_test_client():
+    """
+    Creates the test client we will be using to test the responses
+    from our app, this is a test fixture.
+    :return: A flask test client.
+    """
+    with create_app().test_client() as test_client:
+        yield test_client
+
+
+@pytest.mark.usefixtures("live_server")
+def test_notification_successful_content(flask_test_client):
+    """ GIVEN test check the successful response
```

Comment:
Is this really what it tests? It tests that the word "content" is in the body of the response. Suppose that it gave an error page which stated "Error : No content found" then we would be in trouble. 

We want: 
GIVEN a flask instance running our service
WHEN we make a get request to {{whatever that url is}}
THEN we expect the response to contain the string "content"

Reply:
I went through the list of errors and descriptions, it only spits out content/ body if it is a valid request.

Reply:
https://docs.notifications.service.gov.uk/python.html#error-codes